### DISCUSSION TITLE
Welcome to uecetex2 Discussions!

### DISCUSSION THREAD

**Discussion by thiagodnf** [thiagodnf/uecetex2#26] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Lisraelle02** [comment]
Boa tarde. Estou usando seu template para fazer meu trabalho de mestrado. Ele deu um erro que bugou geral e não estou conseguindo encontrar o erro, Poderia me ajudar?

**Comment by JPAlmeida15** [comment]
Bom dia, também estou usando seu template para fazer meu trabalho de mestrado da UECE, poderia tirar algumas dúvidas sobre seu template??

**Reply by thiagodnf** [reply]
Olá @JPAlmeida15 com certeza vc pode postar aqui as dúvidas

**Reply by JPAlmeida15** [reply]
No arquivo documento.tex na linha 40 ,temos: \ehqualificacao{}

Se coloco **sim**, não imprime a dedicatória, epígrafe e os agradecimentos.
Se coloco **não**,  imprime a dedicatória, epígrafe e os agradecimentos, mas desconfigura a folha de aprovação (o último professor da banca vai para outra página.


Também estou tentando adaptar o template para nova norma da ABNT 10520.

**Reply by thiagodnf** [reply]
> Se coloco sim, não imprime a dedicatória, epígrafe e os agradecimentos.
poderia tentar comentar as seguintes linhas do arquivo uecetex2.sty e ver se funciona?

```tex
\newcommand{\imprimirdedicatoria}[1]{
%	\ifthenelse{\boolean{uecetex2@ehqualificacao}}{}{
		\vspace*{20cm}
		\vfill
		%Retira o espaço do parágrafo
		\hspace*{-2cm}
		%Adiciona o espaço da margem
		\hspace*{8cm}	
		\begin{minipage}{8cm}
			\input{#1}
		\end{minipage}
		\pagebreak
%	}
}
```

> mas desconfigura a folha de aprovação (o último professor da banca vai para outra página.

Quantos professores você está usando? Compilei aqui com 4 e funcionou. 

![image](https://github.com/thiagodnf/uecetex2/assets/114015/3cbb93c1-0243-4e8f-9da5-80b6731ecab5)

**Reply by JPAlmeida15** [reply]
Estou com 5 membros na banca.

**Reply by thiagodnf** [reply]
Adicionei na dissertacao o quinto membro da banca. Poderia baixar e atualizar o arquivo uecetex2.sty?

**Reply by JPAlmeida15** [reply]
Boa tarde professor, atualizei o arquivo uecetex2.sty  e apresentou o seguinte erro: 
![Captura de tela 2023-10-11 141601](https://github.com/thiagodnf/uecetex2/assets/147524640/6f278711-2f9d-47be-93b7-4b5ada706f17)

**Reply by thiagodnf** [reply]
Olá @JPAlmeida15 Acabei de baixar e compilar o projeto e não tive nenhum erro. Poderia dizer se vc ainda tem esse problema?

**Reply by JPAlmeida15** [reply]
Boa noite, desculpe-me pela demora, mas continuou com erro ;(

**Reply by JPAlmeida15** [reply]
Boa tarde, de acordo com o GUIA da UECE (https://drive.google.com/drive/folders/16W1XLu3ixVs9AhaWrRNRDtZOlsTeHy6x)
temos que mudar algumas coisas:

tipo nas listas de ilustrações, tabelas e quadros, tudo tem que tá em negrito.

percebi q ele não inicia a numeração das tabelas pelo número um.

o Sumário tbm não está de acordo com o guia.

estou tentando fazer algumas dessas alterações...

**Comment by thiagodnf** [comment]
Olá @JPAlmeida15,

> tipo nas listas de ilustrações, tabelas e quadros, tudo tem que tá em negrito.

Testei aqui no desktop e no overleaf e tudo tá correto. Pode nos informar qual ambiente você está usando?

<img width="377" alt="image" src="https://github.com/thiagodnf/uecetex2/assets/114015/2cc499e4-c378-4071-bebb-8b0d3b98692b">

<img width="369" alt="image" src="https://github.com/thiagodnf/uecetex2/assets/114015/fa32fcdc-467c-445b-aa9c-3fb8670be373">

> o Sumário tbm não está de acordo com o guia.

Olhei rapidamente também o sumário e também parece correto

> percebi q ele não inicia a numeração das tabelas pelo número um.

A biblioteca foi atualizada recentemente. Você pode baixar o arquivo [uecetex2.sty](https://github.com/thiagodnf/uecetex2/blob/master/lib/uecetex2.sty). Esperamos que esse problema esteja corrigido.

**Reply by JPAlmeida15** [reply]
Olá @thiagodnf  baixei o arquivo uecetex2.sty e apareceu assim quando compilei 
![Captura de tela 2023-10-31 082344](https://github.com/thiagodnf/uecetex2/assets/147524640/0f8bfc38-3f8f-4505-9d42-62ddf976fd43)

e assim: 
![Captura de tela 2023-10-31 082537](https://github.com/thiagodnf/uecetex2/assets/147524640/9fbff9b6-28ec-42a5-b3a4-e75ef8775d8a)


as listas de figuras, tabelas e quadros ficaram em negrito agora...

outro pedido, o pessoal da biblioteca pediu para eu tirar tanto do sumário como do corpo do texto o Anexo, apresentar logo o ANEXO A - [...]

**Comment by thiagodnf** [comment]
@JPAlmeida15 onde você está compilando o seu documento? Overleaf? Windows? Mac? Linux?

**Comment by thiagodnf** [comment]
> outro pedido, o pessoal da biblioteca pediu para eu tirar tanto do sumário como do corpo do texto o Anexo, apresentar logo o ANEXO A - [...]

@JPAlmeida15 poderia baixar novamente o arquivo uecetex2.sty e compilar o seu projeto? Gostaria de saber se esse problema foi resolvido

**Reply by JPAlmeida15** [reply]
@thiagodnf  muito obrigado por tudo, consegui editar o arquivo uecetex2 para o formato q estou precisando, seu template me ajudou bastante. Já enviei o meu trabalho para a biblioteca da UECE.

obs.: estava usando no overleaf

**Comment by danielgleison** [comment]
Boa tarde,
Seguem sugestões de ajustes para adequação ao Guia de Normalização de Trabalhos Acadêmicos da UECE - 5ª edição (2024).

1. No caso de curso à distância, a logo e o nome da Universidade Aberta do Brasil (UAB) devem ser inseridos na capa (pag. 26 do Guia);
2. No sumário, quando o título do apêndice passa para a segunda linha, deve ser recuado à esquerda (pag. 32 do Guia);
3. Incluir a abreviatura de titulação dos orientadores e banca examinadora, conforme padrão da pag. 29 do Guia;
4. Incluir o arquivo pdf da folha de aprovação assinada (foi excluída na última versão).

Consegui ajustar todos os itens manualmente, com exceção do item 2.

**Reply by thiagodnf** [reply]
Olá @danielgleison 

Tentei ajustar o item 2 e também não consegui.
